### PR TITLE
Media gallery terminology fix

### LINKED ISSUE #590: Change "Enhanced Media Gallery" terminology
# Topic clarification/correction request

## Topic Link

Not yet published

## Description

Per our content PM, Olga: "No, please use just new media gallery as we are planning to deprecate old one in next minor release and there will be just “media gallery”

## Possible solutions

Update new topics, content so that we are not declaring Enhanced Media Gallery" as the name of this feature set. Change topic file names also.

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) fixes #590 

There is a new Media Gallery for the 2.40 release to replace the old functionality and will be just "Media Gallery" going forward. We were incorrectly using the name "Media Gallery" for the product images on the storefront. That is now a storefront media browser.

If Magento Commerce, please specify:

- [ ] Commerce only?
- [ ] Commerce with B2B?

## Affected documentation pages

Some pages are new for 2.40 and not yet published in production, so using beta environment

- https://docs-beta.magento.com/user-guide/catalog/media-gallery.html (renamed)
- https://docs-beta.magento.com/user-guide/cms/adobe-stock-license-image.html
- https://docs-beta.magento.com/user-guide/cms/adobe-stock-manage.html
- https://docs-beta.magento.com/user-guide/cms/adobe-stock-save-preview.html
- https://docs-beta.magento.com/user-guide/cms/content-menu.html
- https://docs-beta.magento.com/user-guide/cms/enhanced-media-gallery-asset-management.html (renamed)
- https://docs-beta.magento.com/user-guide/cms/enhanced-media-gallery-folder-management.html (renamed)
- https://docs-beta.magento.com/user-guide/cms/enhanced-media-gallery.html (renamed)
- https://docs-beta.magento.com/user-guide/configuration/advanced/system.html
- https://docs-beta.magento.com/user-guide/stores/attributes-input-types.html

## Links to Magento source code or PRs

https://github.com/magento/merchdocs/pull/507

## Additional information

Internal staging build 519

### CODE REVIEW

File: src/catalog/media-browser.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 
 ![]({% link images/images/storefront-thumbnail-gallery.png %}){: .zoom}
-_Media Gallery_
+_Media Browser_
 
 ## Image zoom
 
-Customers can view a magnified portion of the image on mouseover, if the [base image]({% link catalog/product-image-upload.md %}) is large enough to create the zoom effect. When zoom is activated, you can click the main image and move the cursor around to magnify different parts of the image. The magnified selection appears to the right of the image.
+Customers can view a magnified portion of the image on mouseover, if the [base image]({% link catalog/product-image-upload.md %}) is large enough to create the zoom effect. When zoom is activated, customers can click the main image and move the cursor around to magnify different parts of the image. The magnified selection appears to the right of the image.
```
Comment (hickskenh) [inline src/catalog/media-browser.md:14]:
```suggestion
If the [base image]({% link catalog/product-image-upload.md %}) is large enough to create the zoom effect, customers can view a magnified portion of the image on mouseover. When zoom is activated, customers can click the main image and move the cursor around to magnify different parts of the image. The magnified selection appears to the right of the image.
```

File: src/catalog/search-configuration.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Search suggestions and recommendations can impact server performance.
 
 ## Step 4: Configure Minimum Terms to Match
 
-To control the minimum number of terms from your query that the search results should match in order to be returned, specify a value in the **Minimum Terms to Match** field. This ensures optimal results relevancy for shoppers. Refer to the [Elasticsearch section of Catalog]]({% link catalog.md %}#elasticsearch) for a list of accepted values for this field.
+To control the minimum number of terms from your query that the search results should match in order to be returned, specify a value in the **Minimum Terms to Match** field. This ensures optimal results relevancy for shoppers. For a list of accepted values for this field, see  [minimum_should_match parameter](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-minimum-should-match.html) in the Elasticsearch documentation.
```
Comment (hickskenh) [inline src/catalog/search-configuration.md:93]:
```suggestion
To control the minimum number of terms from your query that the search results should match in order to be returned, specify a value for **Minimum Terms to Match**. This ensures optimal results relevancy for shoppers. For a list of accepted values, see  [minimum_should_match parameter](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-minimum-should-match.html) in the Elasticsearch documentation.
```

File: src/cms/content-menu.md [hunk 3]
```diff
@@ -39,10 +39,10 @@ Create rich, [interactive content]({% link cms/dynamic-blocks.md %}) that is dri
 
 ## Media
 
-[Enhanced Media Gallery]({% link cms/enhanced-media-gallery.md %}) helps Merchants to organize and manage their media files in the admin panel.
+[Media Gallery]({% link cms/media-gallery.md %}) helps Merchants to organize and manage their media files in the admin panel.
```
Comment (hickskenh) [inline src/cms/content-menu.md:42]: most references have "new media gallery".    is there a plan to remove "new" for subsequent releases?  is it necessary now?

Comment (leslietilling) [reply src/cms/content-menu.md:42]: We will remove "new" later after they remove the "old" functionality. Olga will let us know when she wants to make that transition. On this one, the ""new" isn't part of the formal name and it seemed weird to add lowercase "new".

**Pull request merged**